fix: Failed to get likes Error: Failed to get likes: 400

4:31:04 PM [vite] (client) [console.error] [2026-03-25 18:31:04.663] [ERROR] Failed to get likes Error: Failed to get likes: 400
    at handle_like_response (http://localhost:5173/src/lib/api/like-api.ts:11:9)
    at Object.get (http://localhost:5173/src/lib/api/like-api.ts:21:15)
    at async Module.track_reactivity_loss (http://localhost:5173/node_modules/.vite/deps/runtime-DQf5Q2rO.js?v=7c049fbf:2846:14)
    at async #fetch_likes (http://localhost:5173/src/lib/hooks/Lik

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
 	"name": "joshuafolkken-com",
 	"private": true,
-	"version": "0.142.0",
+	"version": "0.143.0",
 	"type": "module",
 	"packageManager": "pnpm@10.32.1+sha512.a706938f0e89ac1456b6563eab4edf1d1faf3368d1191fc5c59790e96dc918e4456ab2e67d613de1043d2e8c81f87303e6b40d4ffeca9df15ef1ad567348f2be",
 	"engines": {

--- a/src/lib/utils/slug-validator.ts
+++ b/src/lib/utils/slug-validator.ts
@@ -1,5 +1,6 @@
 const SLUG_MAX_LENGTH = 200
-const SLUG_PATTERN = /^[\w-]+$/u
+// Word chars, hyphens, and colons (e.g. page:projects for static-page likes).
+const SLUG_PATTERN = /^[\w:-]+$/u
 
 function parse_slug(value: unknown): string | undefined {
 	if (typeof value !== 'string' || value.trim().length === 0) {

